spec/models/qa/authorities/local/find_works_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe Qa::Authorities::Local::FindWorks do
+  subject { described_class.new(q: "Query") }
+
+  describe "#properties" do
+    it { is_expected.to respond_to("q") }
+    it { is_expected.to respond_to("search") }
```

Comment:
Should this spec expand to test more of the `#search` method's behavior, or do you think this is covered well by other parts of the test suite?